src/arrayMethodReduce.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Reduce
  */
 function applyCustomReduce() {
-  [].__proto__.reduce2 = function(callback, initialValue) {
+  [].__proto__.reduce2 = function(callback, initialValue, index = 0) {
```

Comment:
```suggestion
  [].__proto__.reduce2 = function(callback, initialValue = this[0]) {
```
Давай зробимо, щоб наш метод працював так само, як і справжній reduce.
Reduce має приймати два аргумента, а не три.
Змінюй startIndex в залежності від того, скільки аргументів було передано